docker-compose.yml

```diff
@@ -25,12 +25,11 @@ hydroshare:
     SOLR_HOST: solr
   volumes:
     # hydroshare repository
-    - ".:/home/docker/hydroshare"
-    #- "/home/hydro/hydroshare:/home/docker/hydroshare"
+    - "/home/hydro/hydroshare:/home/docker/hydroshare"
```

Comment:
No need to include this file in the PR.

The `docker-compose.yml` file gets generated at runtime from `docker-compose.template` using the definitions from `config/hydroshare-config.yaml`.